tests/phpunit/tests/filesystem/wpOpcacheInvalidateDirectory.php

```diff
@@ -37,7 +37,7 @@ public static function set_up_before_class() {
 	public function test_should_trigger_error_with_invalid_dir( $dir ) {
 		$this->expectError();
 		$this->expectErrorMessage(
-			'<code>wp_opcache_invalidate_directory()</code>',
+			'wp_opcache_invalidate_directory(): <code>wp_opcache_invalidate_directory()</code> expects a non-empty string.',
```

Comment:
As suggested above, removing the redundant function name.

```suggestion
			'<code>wp_opcache_invalidate_directory()</code> expects a non-empty string.',
```